ts/components/ui/LogoPaymentExtended.tsx

```diff
@@ -11,7 +11,7 @@ type LogoPaymentHugeProps = {
       icon: "payPal" | "bpay";
     }
   | {
-      abiCode: string;
```

Comment:
This type is the same as

```
type LogoPaymentHugeProps = {
  dimensions: { height: number; width: number };
  icon?: "payPal" | "bpay";
  abiCode?: string;
};
```

I think it's more readable. What do you think?

Also, `LogoPaymentHugeProps` was not renamed with the component. I think it should be `LogoPaymentExtendedProps` or, since it is not exported, simply `Props`.

Reply:
right, addressed 👍